### PR TITLE
Use 6 months as SSL STS header threshold

### LINKED ISSUE #23957: Hide security and settings warning
I'm using CloudFlare as shield for my ownCloud installation.
It only allows to set `Strict-Transport-Security` to 6 months max (15552000 seconds instead of desired 15768000). Generally I'm OK with current settings and how CloudFlare does it, but annoying warning keeps showing. I need the ability to hide it.

### PR DESCRIPTION
**Pull request by MorrisJobke**
- this uses 6 months (6 \* 30 \* 24 \* 60 \* 60 = 15552000)
- old value was half a year (365 / 2 \* 24 \* 60 \* 60 = 15768000)
- fixes #23957 

cc @nazar-pc @nickvergessen @rullzer @LukasReschke

### CODE REVIEW

File: core/js/setupchecks.js [hunk 1]
```diff
@@ -273,7 +273,7 @@
 						}
 					}
 
-					var minimumSeconds = 15768000;
+					var minimumSeconds = 15552000;
```
Comment (nickvergessen) [inline core/js/setupchecks.js]: could do the math here, or add a comment about the math

**Comment by rullzer** [comment]
Fine with me.
But this simplification that a month has 30 days is not true :stuck_out_tongue: 

:+1:

**Comment by LukasReschke** [comment]
:+1:

**Comment by nazar-pc** [comment]
:+1:

**Pull request merged**